src/UCTNode.cpp

```diff
@@ -49,9 +49,6 @@ UCTNode::UCTNode(int vertex, float score, float init_eval)
 }
 
 UCTNode::~UCTNode() {
```

Comment:
I'm a C++ noob, is there a reason not to delete the destructor function completely now that it's empty?

Reply:
It's already like that in "next" actually.

~UCTNode() = default;